src/Utilities/Utilities.jl

```diff
@@ -66,6 +66,11 @@ srcpath(source, root, file) = normpath(joinpath(relpath(root, source), file))
 Slugify a string into a suitable URL.
 """
 function slugify(s::AbstractString)
+    stripped_s = lstrip(!isletter,s)
+    # in case there is no letter at all, take the original approach
+    if !isempty(stripped_s)
+        s = stripped_s
+    end
```

Comment:
Is there a particular problematic case that you came across and this tries to resolve?

Reply:
I stumbled across the heading "1) problematic heading " giving me a warning when creating an epub because technically the slugified version starting with a "("  isn't a URL.

Reply:
Technically, parentheses should be valid in URL fragments though?

But we already strip numbers, so I do agree that we should strip various other non-letters. But we probably want to keep e.g. `@` and `_`, as they are valid variable names in Julia.

Reply:
Yes, they are allowed, but in this context the slugified target must match the id-attribute and for those I followed the recommendation from: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id